ContributeDocumentation.md

```diff
@@ -0,0 +1,88 @@
+# Contributing Application Documentation Changes
+
+This document describes how to contribute changes to the ISIS Application Documentation, which is available online for the most recent ISIS release at: https://isis.astrogeology.usgs.gov/Application/index.html
+
+# Getting Started
+Documentation for ISIS applications is located within the ISIS source code and written in XML. The XML is processed to create the documentation that can be viewed on the ISIS website at: https://isis.astrogeology.usgs.gov/Application/index.html
```

Comment:
This makes it sound like the docs are embedded in the cpp files. Suggest reword to say it is in the directory for each app with the same name except .xml